src/libcmd/command.cc

```diff
@@ -34,6 +34,18 @@ nlohmann::json NixMultiCommand::toJSON()
     return MultiCommand::toJSON();
 }
 
+void NixMultiCommand::run()
+{
+    if (!command) {
+        std::set<std::string> subCommandNames;
+        for (auto & [name, _] : commands)
+            subCommandNames.insert(name);
+        throw UsageError("'nix %s' requires a sub-command.\nAvailable sub-commands: %s",
+                commandName, concatStringsSep(", ", subCommandNames));
```

Comment:
I've done this the easy way and just let `lowdown` handle everything in 525501e9d. That gives us a nice list (and better highlighting since each command is now individually highlighted rather than having the whole list highlighted at once)

Reply:
Nice!